scripts/fetch/fetchGA.R

```diff
@@ -15,12 +15,11 @@ fetch.GAviews <- function(viz) {
     viz[['fetcher']] <- 'sciencebase'
     viz[['remoteFilename']] <- basename(viz[['location']])
     message("Downloading SB file...")
-    fetch(as.fetcher(viz))
+    #fetch(as.fetcher(viz))
     message('Downloaded SB file')
 
     if(viz[['update']]) {
-      masterTable <- readDepends(viz)[['project_table']]
-
+      masterTable <- do.call(bind_rows, readDepends(viz)[['project_table']])
```

Comment:
We could put a custom reader on 'project_table' so we don't have to remember to `bind_rows` when reading it.